src/Lock.php

```diff
@@ -73,27 +73,17 @@ public static function showForItem(CommonDBTM $item)
             return false;
         }
 
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
```

Comment:
Should not this messages be hidden if there are no locked fields (condition in iterator count)?